app/assets/javascripts/discourse/app/components/topic-timeline/back-button.js

```diff
@@ -0,0 +1,3 @@
+import GlimmerComponent from "discourse/components/glimmer";
+
+export default class TopicTimelineBackButton extends GlimmerComponent {}
```

Comment:
If there is no state/js required in a Glimmer component, then we can save the performance cost of initializing a component instance. To do that, we can use the `templateOnly()` helper:

```suggestion
import templateOnly from "@ember/component/template-only";

export default templateOnly();
```